src/TestRunnerOrchestrator.ts

```diff
@@ -168,7 +168,7 @@ export default class TestRunnerOrchestrator {
   }
 
   private createTestRunnerSandboxes(): Promise<TestRunnerSandbox[]> {
-    let cpuCount = os.cpus().length;
+    let cpuCount = this.options.cpuCount || os.cpus().length;
```

Comment:
I actually would like `maxConcurrency` or `concurrency` better. That way, Stryker can still choose not to spin up more test runners than it needs. For example, why spin up 8 test runners for 6 tests? 

The default would be `infinite`. That way you let Stryker decide what's best.

@jeznag